tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -1373,6 +1373,45 @@ Status Converter::RenameAndMarkOutputTensors(
   return Status::OK();
 }
 
+#if IS_TRT_VERSION_GE(7, 1, 3, 0)
+// Force TensorRT deterministic behavior by enforcing
+// TRT tactic selection to be a specific one controlled by the env var:
+// TF_DEBUG_TRT_DETERMINISTIC_TACTIC_ID.
```

Comment:
Let't revise the comment for this class to this:

An algorithm selector that always returns a specific ID for selectAlgorithms. This is used to support the implementation of using environment variable TF_TRT_ALGORITHM_ID for debugging TensorRT.

Reply:
Done